Algorithm.Framework/Portfolio/BlackLittermanOptimizationPortfolioConstructionModel.py

```diff
@@ -192,7 +192,7 @@ def OnSecuritiesChanged(self, algorithm, changes):
             if not history.empty:
                 ticker = SymbolCache.GetTicker(symbol)
 
-                if ticker not in history.index.levels[0]:
+                if ticker  not in history.index.levels[0]:
```

Comment:
Glad this is working now -- looks like there's a double space following `ticker`